Modules/Core/Common/include/itkRegion.h

```diff
@@ -90,6 +90,13 @@ class ITKCommon_EXPORT Region
   Region() = default;
   virtual ~Region() = default;
 
+  Region(const Region &) = default;
```

Comment:
Would "rule of zero" work here, but removing all of them?

Reply:
@dzenanz  Good question... technically, the rule of zero would probably work as well, for `Region` but then we would also have to remove `~Region()`, which is now declared `virtual`. And you know, `Region` is still being used as base class. And a base class without virtual destructor, while still having (other) virtual functions looks like a code smell... at least it might cause compiler warnings  😸

Reply:
So the short answer is: no, I don't think "rule of zero" works for `itk::Region`. 🤷  The current PR should work fine, though 👍 And it's not even an API change, because the added (defaulted) member functions behave exactly as in the original code.

Reply:
Thanks for the comments. Resolving. Ready to be merged.